BackofficeBundle/Resources/public/coffee/page/showTemplate.coffee

```diff
@@ -6,7 +6,11 @@ showTemplate = (url)->
     success: (response) ->
       template = new Template
       template.set response
-      view = new TemplateView(template: template)
+      view = new TemplateView(
+        template: template
+        extendView : [ 'commonPage', 'addArea' ]
```

Comment:
No, we can't because of the reccursive merge of extendView before the initialize in orchestrView constructor.